Create ui-c8y-1021-60-1-fixed-issue-with-last-updated-field-not-shown-backport-release-cd.md (#3208)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-60-1-fixed-issue-with-last-updated-field-not-shown-backport-release-cd.md

---
date: ""

> title: Fixed issue with lastUpdated field in the alarm details page being empty.

Reviewer: ```suggestion
title: Last occurrence of an alarm now properly shown in alarm details
```

> title: Fixed issue with lastUpdated field not shown. BACKPORT (release/cd) (#8856)

Reviewer 2: ```suggestion
title: Fixed issue with lastUpdated field in the alarm details page being empty.
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62788
version: 1021.60.1
---

> Fixed issue with lastUpdated field not shown. BACKPORT (release/cd) (#8856)

Reviewer 2: ```suggestion
In the alarm details view, the lastUpdated field was empty which usually shows the time of the last alarm occurrence. This issue has been fixed and the lastUpdated field has been renamed to "Last occurrence" instead. Users can see again when an alarm has been updated by looking at the "Last occurrence" field in the alarm details view.
```

> In the alarm details view, the lastUpdated field was empty which usually shows the time of the last alarm occurrence. This issue has been fixed and the lastUpdated field has been renamed to "Last occurrence" instead. Users can see again when an alarm has been updated by looking at the "Last occurrence" field in the alarm details view.

Reviewer: ```suggestion
In the alarm details view, the **Last updated** field, which usually shows the time of the last alarm occurrence, was empty. This issue has been fixed. Moreover, the field has been renamed to **Last occurrence**. 
```
